src/Symfony/Component/Security/Http/Authenticator/Passport/Passport.php

```diff
@@ -21,7 +21,7 @@
  *
  * @author Wouter de Jong <wouter@wouterj.nl>
  */
-class Passport implements UserPassportInterface
+class Passport implements UserPassportInterface, AttributedPassportInterface
```

Comment:
Please change the doc blocks of `setAttribute()` and `getAttribute()` to…

```php
/**
 * {@inheritdoc}
 */
```

Reply:
Done